docs/integration.rst

```diff
@@ -65,3 +65,44 @@ You can configure yamllint to run on ``arc lint``. Here is an example
       }
     }
   }
+
+Intergration with GitLab
+------------------------
+
+You can use the following gitlab-ci stage to do run yamllint and get the results as a
+`Code quality (Code Climate) <https://docs.gitlab.com/ee/ci/testing/code_quality.html>` report.
+
+.. code:: yaml
+
+   ---
```

Comment:
Code snippets after a line `.. code:: yaml` should be indented of 1 space (not 3). I know other examples in this file use 2 or 3 spaces, but for this new one could you use just 1? Thanks

Reply:
Sure, fixed the others as well.

Reply:
Thanks :+1: